tests/fsharp/typecheck/sigs/neg90.fs

```diff
@@ -13,4 +13,17 @@ module First =
   [<RequireQualifiedAccess>]
   type DU = Member of int
 
-let _ = First.Member(0)
\ No newline at end of file
+let _ = First.Member(0)     // compiles, but should not
+let _ = First.DU.Member(0) // correct
+
+// See https://github.com/Microsoft/visualfsharp/issues/95 - part 2
+module ModuleWithRecord =
+    [<RequireQualifiedAccess>]
+    type Record1 = { Field1 : int }
+
+let _ = { ModuleWithRecord.Record1.Field1 = 42 } // correct
+
+open ModuleWithRecord
+let _ = { Record1.Field1 = 42 }                  // correct
+let _ = { ModuleWithRecord.Field1 = 42 }         // compiles, but should not
+let _ = { ModuleWithRecord.Record1.Field1 = 42 } // correct
```

Comment:
add newline at end of file, is good for textfiles and harmless for test behaviour